### DISCUSSION TITLE
Does the development kit support sync to Insights Hub and Senseye?

### DISCUSSION THREAD

**Discussion by tobsie16** [industrial-edge/iih-essentials-development-kit#23] [Q&A] [answered]
Does the development kit support sync to Insights Hub and Senseye without having an Ind. Edge device?

**Comment by KohlerL-Smns** [comment, marked as answer]
Yes. All data stored within IIH Essentials (either via Simulation or via API) can be synced to both the available destinations, Insights Hub and Senseye.
